mysql-test/suite/innodb/t/percona_doublewrite.test

```diff
@@ -138,6 +138,64 @@ SET GLOBAL innodb_fast_shutdown=2;
 --let $restart_parameters=restart:--innodb-force-recovery=6
 --source include/start_mysqld.inc
 
+--echo #
+--echo # Bug PS-3411: Parallel doublewrite buffer file created when skip-innodb_doublewrite is set
+--echo #
+
+--let $restart_hide_args= 1
+--let $restart_parameters= restart:--innodb-parallel-doublewrite-path=$DOUBLEWRITE_FILE
+--source include/restart_mysqld.inc
+file_exists $DOUBLEWRITE_FILE;
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+--enable_warnings
+
+CREATE TABLE t1 (a INT);
+
+--let $i= 1000
+--disable_query_log
+START TRANSACTION;
+while($i)
+{
+    --eval INSERT INTO t1 VALUES ($i);
+    --dec $i
+}
+COMMIT;
+--enable_query_log
+
+# flush is done once a second so wait to be sure that we test what we need
+--real_sleep 3
```

Comment:
Sleep (even real_sleep) can be unreliable on jenkins. It's better to use wait_condition / debug sync points, depending on what's the easiest to do for a given situation. For page flushes, you probably want somethning like

```
# wait until pages flushed
--let $wait_condition= SELECT variable_value = 0 FROM performance_schema.global_status WHERE LOWER(variable_name) = 'innodb_buffer_pool_pages_dirty'
--source include/wait_condition.inc
```